.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1472,13 +1471,19 @@ public class ChatMessage : IJsonModel<ChatMessage>, IPersistableModel<ChatMessag
         public static UserChatMessage CreateUserMessage(params ChatMessageContentPart[] contentParts);
         public static UserChatMessage CreateUserMessage(IEnumerable<ChatMessageContentPart> contentParts);
         public static UserChatMessage CreateUserMessage(string content);
-        public static implicit operator ChatMessage(string userMessage);
+        public static implicit operator ChatMessage(string content);
         ChatMessage IJsonModel<ChatMessage>.Create(ref Utf8JsonReader reader, ModelReaderWriterOptions options);
         void IJsonModel<ChatMessage>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options);
         ChatMessage IPersistableModel<ChatMessage>.Create(BinaryData data, ModelReaderWriterOptions options);
         string IPersistableModel<ChatMessage>.GetFormatFromOptions(ModelReaderWriterOptions options);
         BinaryData IPersistableModel<ChatMessage>.Write(ModelReaderWriterOptions options);
     }
+    public class ChatMessageContent : ObjectModel.Collection<ChatMessageContentPart> {
+        public ChatMessageContent();
+        public ChatMessageContent(params ChatMessageContentPart[] parts);
+        public ChatMessageContent(IList<ChatMessageContentPart> parts);
```

Comment:
In this constructor, I take an `IList` instead of an `IEnumerable` because `Collection<T>`'s constructor takes an `IList`.

For consistency, I think I should also update the constructors of the different message types to take an `IList` instead of an `IEnumerable`. 

Thoughts?

Reply:
If what you want is to have the ability to foreach over parts, maybe just implement `IEnumerable<T>` instead of `Collection<T>`